FrontendApartmentReservation/src/actions/index.jsx

```diff
@@ -13,74 +15,97 @@ var headers = {
 
 export const getAllApartments = () => (dispatch) => {
     console.log("index")
-    return fetch(`https://localhost:44334/api/apartments`, {
+    return fetch(`https://backend-apartments-kibanaprod.1d35.starter-us-east-1.openshiftapps.com/api/apartments`, {
```

Comment:
Please dont have urls in github. They must be stored as environment variables on firebase or wherever they are deployed.

Also, for you not to have to use same url in all the requests, have some sort of a variable as a base, e.g. `BACKEND_BASE_URI`